src/db/seeders/20230226163601-demo-data.js

```diff
@@ -20,10 +20,15 @@ module.exports = {
 
       await queryInterface.bulkInsert('Units', [{
         name: 'Spearman',
-        attackPower: 3,
-        armor: 1,
         maxHealth: 4,
         movementSpeed: 40,
+        pointsCost: 10,
+        armorSaveRoll: 5,
+        meleeNumAttacks: 3,
+        meleeHitRoll: 3,
+        meleeWoundRoll: 4,
+        meleeArmorPiercing: 1,
+        meleeDamage: 2,
```

Comment:
No worries here about modifying an existing seeder. See [here](https://sequelize.org/docs/v6/other-topics/migrations/#running-seeds) in the sequelize documentation where they explain:

> Note: Seeder execution history is not stored anywhere, unlike migrations, which use the SequelizeMeta table. If you wish to change this behavior, please read the Storage section.

This means you can just undo and re-run a seeder to get the new data, so it's fine to just modify this one rather than creating a new one.